Packs/Expanse/Playbooks/Expanse_Incident_Playbook_CHANGELOG.md

```diff
@@ -3,4 +3,4 @@
 
 ## [20.2.4] - 2020-02-25
 #### New Playbook
-Parses incident from Expanse in raw json to readable output
\ No newline at end of file
+Parses incident from Expanse in raw json to readable output
```

Comment:
?

Reply:
Couldn't understand what the diff here is

Reply:
Me too.